src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy

```diff
@@ -61,7 +61,7 @@ class DirectSynchronousExecutionJobManager extends BatchEuphoriaJobManager<Direc
 
     @Override
     String getJobIdVariable() {
-        return '$'
+        return BEJob.PARM_JOBCREATIONCOUNTER
```

Comment:
Unfortunately, the value of the counter is not the jobIDVariable. The current solution breaks the semantics of the method.